src/pymortests/vectorarray.py

```diff
@@ -266,6 +266,29 @@ def test_copy(vectors_and_indices):
             pass
 
 
+@pyst.given_vector_arrays(index_strategy=pyst.valid_indices)
```

Comment:
Isn't this part of a separate PR?

Reply:
It started here and I branched it out to #1419, which I thought would be merged earlier...

Reply:
will rebase on main once #1419 is in, keeping this unresolved as a reminder

Reply:
done